### PR TITLE
adding private_key_jwt as an example

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** adding private_key_jwt as an example
- **Is this PR related to a Monolith release?** no

### Resolves:

* [OKTA-623222](https://oktainc.atlassian.net/browse/OKTA-623222)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/api/oidc/index.md](https://github.com/okta/okta-developer-docs/blob/fe3fd33a3a50beda1d2dd4cf75f4ca34ecfb45e4/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Foidc%2Findex.md) | 70 | 555 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/eed54429-31c7-499b-9cf3-f7cc5da39878) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 1 of 1 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.1630325d-7c44-46d0-a48e-cf7b1eec4476?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

File: packages/@okta/vuepress-site/docs/reference/api/oidc/index.md [hunk 1]
```diff
@@ -665,15 +665,31 @@ The following parameters can be posted as a part of the URL-encoded form values
...
+-d "token_type_hint=access_token"
+```
+
+#### Request example (access token, using `private_key_jwt` client authentication)
+
+```bash
+curl -v -X POST \
+-H "Content-Type: application/x-www-form-urlencoded" \
+-H "User-Agent: Mozilla/5.0 (${systemInformation}) ${platform} (${platformDetails}) ${extensions}" \
+'https://sharper.trexcloud.com/oauth2/v1/introspect'
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/reference/api/oidc/index.md:688]:
```suggestion
'https://example.okta.com/oauth2/v1/introspect'
```

**Review by brianduffield-okta — APPROVED** [review]
LGTM, one nit

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/api/oidc/index.md](https://github.com/okta/okta-developer-docs/blob/c313502631c23b852e8c607e2983b6a3ef07dbc6/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Foidc%2Findex.md) | 70 | 555 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/85f09950-6965-424d-955c-52b28e8fb0ae) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 1 of 1 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.4892be61-0d35-4c4f-abdb-05467e772618?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Review by barbaravo-okta — APPROVED** [review]

**Pull request merged**